src/clp_logging/handlers.py

```diff
@@ -59,12 +58,8 @@ def _init_timeinfo(fmt: Optional[str], tz: Optional[str]) -> Tuple[str, str]:
     if not fmt:
         fmt = "yyyy-MM-d H:m:s.A"
     if not tz:
-        tzf: Optional[tzinfo] = dateutil.tz.gettz()
-        if tzf:
-            tzp: Path = Path.resolve(Path(tzf._filename))  # type: ignore
-            tz = "/".join([tzp.parent.name, tzp.name])
-        else:
-            tz = "UTC"
+        tz = tzlocal.get_localzone_name()
```

Comment:
from the official doc I didn't see examples w.r.t. error handling. But from the implementation details it seems like it could raise an exception or return None (according to their comments). Shall we add a try block to catch any exceptions?

Reply:
I agree. In fact when I was looking for a Python 3.7 compatible version of the lib, I found this from their docs:

https://pypi.org/project/tzlocal/5.1/#:~:text=tzlocal%204.0%20also%20adds%20an%20official%20function%20get_localzone_name()%20to%20get%20only%20the%20timezone%20name%2C%20instead%20of%20a%20timezone%20object.%20On%20unix%2C%20it%20can%20raise%20an%20error%20if%20you%20don%E2%80%99t%20have%20a%20timezone%20name%20configured%2C%20where%20get_localzone()%20will%20succeed%2C%20so%20only%20use%20that%20if%20you%20need%20the%20timezone%20name.

> tzlocal 4.0 also adds an official function get_localzone_name() to get only the timezone name, instead of a timezone object. On unix, it can raise an error if you don’t have a timezone name configured, where get_localzone() will succeed, so only use that if you need the timezone name.